### ISSUE TITLE
Angular 2

### ISSUE THREAD

**Issue by niesteszeck** [creativetimofficial/material-bootstrap-wizard#19]
Hi,

Any plan to convert it to angular 2 without jquery?

**Comment by alexandru-paduraru** [comment]
@niesteszeck yes, we will update it to Angular 2+, as soon as we finish to convert it to Bootstrap 4. This should happen in January 2018. Best, Alex!

**Comment by niesteszeck** [comment]
Hi @alexandru-paduraru I have some progress in this, I'm preparing a repo. I'll let you know

**Comment by niesteszeck** [comment]
Hi @alexandru-paduraru [here](https://github.com/niesteszeck/ng2-material-bootstrap-wizard) is my proposal

**Comment by alexandru-paduraru** [comment]
@niesteszeck that's great man! My colleague @chelaruc will have a look and give feedback. If everything is fine, we will point our users to it.
